### DISCUSSION TITLE
add option to lock difficulty customization mod to values above/below the beatmap settings

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#13100] [Ideas]
this will give to option to lock the DC mod to being the value above or below the beatmap setting

an example:
1. i go to a beatmap with AR 8
2. i use the DC mod to set it to AR 9 (beatmap AR+1)
3. i lock the AR for the DC mod
4. i go to a beatmap with AR 6
5. the DC mod automatically sets it to the beatmap AR+1 which is AR 7 

you can lock the setting for each beatmap attribute individually  

i suggest this to make it less annoying when you want a beatmap attribute above or below the beatmap setting so that you don't have the change the settings everytime you go to another beatmap

for the cases where you can't move the beatmap setting out (moving the setting higher when the value of the setting is already 10), it will not move the setting but it will keep the increased/decreased value that it had before

**Comment by peppy** [comment]
if anything, this would be a setting on all speed adjust mods called something like "scale AR/OD to be unaffected".
